packages/core/src/components/accordion/accordion.tsx

```diff
@@ -1,7 +1,8 @@
-import { h, Component, ComponentInterface, Prop, Host, Method, Element } from "@stencil/core";
-import { AccordionInternalState, AccordionVariant } from "./accordion.interfaces";
+import { Component, ComponentInterface, Element, h, Host, Method, Prop } from "@stencil/core";
```

Comment:
Ik verwacht dat case sensitive sorting ervoor zorgt dat upper en lower case gegroepeerd gaat worden:

```ts
import { h, Component, ComponentInterface, Element, Host, Method, Prop } from "@stencil/core";
import { Component, ComponentInterface, Element, Host, Method, Prop, h } from "@stencil/core";
```

Deze comment ter reminder voor mezelf om dit bij de volgende review te checken

Reply:
Met de nieuwste configuratie, met name `caseInsensitive: false` is het geworden:

```ts
import { Component, ComponentInterface, Element, Host, Method, Prop, h } from "@stencil/core";
```